tests/unit/classifier/svm/LibLinear_unittest.cc

```diff
@@ -206,13 +191,12 @@ class LibLinear : public ::testing::Test
 	}
 	void generate_data_l1_simple()
 	{
-			initialize_train_and_test_and_ground_simple();
-
-			transpose_feats_matrix();
-	}
-	void generate_data_l2_simple()
-	{
-			initialize_train_and_test_and_ground_simple();
+		generate_data_l2_simple();
+		//transpose train_feats matrix
```

Comment:
the comment is useless ... super minor :)

Reply:
Removed it :)